cluster/gce/ubuntu/node.yaml

```diff
@@ -0,0 +1,281 @@
+From nobody Tue Aug 11 10:13:54 2015
+Content-Type: multipart/mixed; boundary="===============6024533374511606659=="
+MIME-Version: 1.0
+
+--===============6024533374511606659==
+MIME-Version: 1.0
+Content-Type: text/upstart-job; charset="us-ascii"
+Content-Transfer-Encoding: 7bit
+Content-Disposition: attachment; filename="kube-addons.conf"
+
+#upstart-job
```

Comment:
They are not completely sequentially executed. The dependency is defined in the line "start on" of each job. The whole sequence is as follows:
(1) kube-env.conf and kube-install-packages.conf (they are in parallel);
(2) kube-install-minion.conf is after kube-env.conf as it sources the env variables there;
(3) kubelet.conf and kube-proxy are after kube-install-packages.conf and kube-install-minion.conf. They are in parallel;
(4) kube-docker.conf is after kubelet.conf;
(5) kube-addons.conf is after kube-docker.conf